sdk/storage/storage-blob/rollup.base.config.js

```diff
@@ -90,9 +90,14 @@ export function browserConfig(test = false, production = false) {
           "if (isNode)": "if (false)"
         }
       }),
-      // os is not used by the browser bundle, so just shim it
+      // fs and os are not used by the browser bundle, so just shim it
+      // dotenv doesn't work in the browser, so replace it with a no-op function
       shim({
         dotenv: `export function config() { }`,
+        fs: `
```

Comment:
I assume this is to resolve the warnings from rollup, is that the case ?

Reply:
Yes, unresolved dependency for browser roll up.